kie-maven-plugin/src/it/kie-maven-plugin-test-kjar-8-exec-model/kie-maven-plugin-test-kjar-8-modA-exec-model/src/test/java-filtered/org/kie/maven/plugin/ittests/MultiModuleTestIT.java

```diff
@@ -1,68 +1,55 @@
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.kie.maven.plugin;
 
+package org.kie.maven.plugin.ittests;
+
+import java.io.File;
+import java.net.URL;
+import java.net.URLClassLoader;
 import java.util.Collection;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
+
 
-import io.takari.maven.testing.executor.MavenRuntime;
+
+import org.assertj.core.api.Assertions;
 import org.drools.compiler.kie.builder.impl.KieContainerImpl;
 import org.junit.Test;
 import org.kie.api.KieServices;
-import org.kie.api.builder.ReleaseId;
 import org.kie.api.definition.KiePackage;
 import org.kie.api.definition.type.FactType;
+import org.kie.api.runtime.KieContainer;
 
 import static java.util.Arrays.asList;
 import static java.util.stream.Collectors.toList;
 import static org.assertj.core.api.Assertions.*;
 
-public class MultiModuleTest extends KieMavenPluginBaseIntegrationTest {
+public class MultiModuleTestIT {
 
     private final static String GROUP_ID = "org.kie";
-    private final static String ARTIFACT_ID = "kie-maven-plugin-test-kjar-8-modA";
-    private final static String VERSION = "1.0.0.Final";
-    private final static String KJAR_NAME = "kjar-8-multimodule";
-
-    public MultiModuleTest(MavenRuntime.MavenRuntimeBuilder builder) {
-        super(builder);
-    }
-
-    @Test
-    public void testMultiModuleWithExecutableModel() throws Exception {
-        testMultiModuleExec(true);
-    }
+    private final static String GAV_ARTIFACT_ID = "kie-maven-plugin-test-kjar-8-modA-exec-model";
+    private static final String GAV_VERSION = "${org.kie.version}";
 
     @Test
     public void testMultiModule() throws Exception {
-        testMultiModuleExec(false);
-    }
-
-    public void testMultiModuleExec(boolean executableModel) throws Exception {
-        if (executableModel) {
-            buildKJarProject(KJAR_NAME, new String[]{"-Dorg.kie.version=" + TestUtil.getProjectVersion()}, "clean", "install");
-        } else {
-            buildKJarProject(KJAR_NAME, new String[]{"-Dorg.kie.version=" + TestUtil.getProjectVersion(), "-DgenerateDMNModel=NO"}, "clean", "install");
-        }
-
-        KieContainerImpl kContainer = null;
-        KieServices kieServices = KieServices.Factory.get();
-        ReleaseId releaseId = kieServices.newReleaseId(GROUP_ID, ARTIFACT_ID, VERSION);
+        final URL targetLocation = MultiModuleTestIT.class.getProtectionDomain().getCodeSource().getLocation();
+        KieContainer kContainer =  null;
         try {
-            kContainer = (KieContainerImpl) kieServices.newKieContainer(releaseId);
+            kContainer = (KieContainerImpl) ITTestsUtils.getKieContainer(targetLocation, GAV_ARTIFACT_ID, GAV_VERSION);
```

Comment:
@mariofusco 
Now with this change the kieContainer is loaded as `getKieClasspathContainer` instead of `newKieContainer` + releaseId. Are these methods equivalent? Do you expect any possible different behaviour?

Reply:
@danielezonca @mariofusco 
As per private conversation, that replacement should be transparent regarding actual execution. Scope of these tests is to verify the created kjars in a kiecontainer, and not the way they are loaded inside it.